src/datasets/tasks/question_answering.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+from typing import Dict
+
+from ..features import Features, Sequence, Value
+from .base import TaskTemplate
+
+
+@dataclass(frozen=True)
+class QuestionAnswering(TaskTemplate):
+    task = "question-answering"
+    input_schema = Features({"question": Value("string"), "context": Value("string")})
```

Comment:
Maybe you want to check with a few QA datasets that this schema make sense. Typically NaturalQuestions, TriviaQA and can be good second datasets to compare to and be sure of the generality of the schema.

A good recent list of QA datasets to compare the schemas among, is for instance in the UnitedQA paper: https://arxiv.org/abs/2101.00178

Reply:
thanks for the tip! added to #2371